File: 1791-richest-customer-wealth/richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int max=Integer.MIN_VALUE;
        for(int i=0;i<accounts.length;i++){
            int count=0;
            for(int j=0;j<accounts[i].length;j++){
                count+=accounts[i][j];
            }
            max=Math.max(max,count);
        }
        return max;
    }
}

File: 1791-richest-customer-wealth/richest-customer-wealth.py
class Solution(object):
    def maximumWealth(self, accounts):
       mc=0
       for i in range(0,len(accounts)):
        c=0
        for j in range(0,len(accounts[i])):
            c+=accounts[i][j]
        if(c>mc):
            mc=c
       return mc

        